Yoast/wpseo-news
# CS/QA: Rename class files

## Summary

This PR can be summarized in the following changelog entry:

* _N/A_

## Relevant technical choices:

As discussed with and approved by @herregroen.

Adjusts the file names to be in line with the rules as laid out in the `Yoast.Files.FileName` sniff.
Ref: Yoast/yoastcs#48

* Remove `class-` prefixes from the plugin file names.
* Make sure that the file names accurately reflect the class names (excluding the prefixes which have been set at `(yoast_)wpseo_news_`).

Where necessary, adjusted any `include`/`require`s to point to the new file names.

Includes adjusting the PHPUnit config file, removing the file name `prefix` attribute and adjusting the `suffix` attribute.


## Test instructions

This PR can be tested by following these steps:

* Check out this branch.
* Run `composer dump-autoload`
* See if the plugin still loads & functions as expected etc.
